bin/dfx-sns-demo

```diff
@@ -91,5 +91,7 @@ sleep 1
 ./bin/dfx-sns-disburse-neuron --network "$DFX_NETWORK"
 sleep 1
 ./bin/dfx-sns-transfer-to-faucet --network "$DFX_NETWORK"
+sleep 1
+./bin/dfx-ii-create-dummy-anchor --network "$DFX_NETWORK"
```

Comment:
I wasn't sure if this should to here or only in the workflow after executing the demo.